jf_agent/main.py

```diff
@@ -523,14 +524,18 @@ def get_issues_to_scan_from_jellyfish(config, creds, updated_within_last_x_month
         data = resp.json()
         print(data.get('message', ''))
     except json.decoder.JSONDecodeError:
-        print(
-            f'ERROR: Could not parse response with status code {resp.status_code}. Contact an administrator for help.'
+        agent_logging.log_and_print_error_or_warning(
+            logger,
+            logging.ERROR,
+            msg_args=[resp.status_code],
+            error_code=3001,
+            exc_info=True,
```

Comment:
This mode is kind of an exception case, and I don't think writes to the logger, so fine to leave this as a `print`